docs/src/style.css

```
@@ -67,3 +67,132 @@ html:not(.dark) #banner a {
+  position: relative;
+  width: 1.15em;
+  height: 1.15em;
+  flex: 0 0 auto;
+}
+
+.notte-fast-setup__button-icon::before,
+.notte-fast-setup__button-icon::after {
+  position: absolute;
+  width: 0.62em;
+  height: 0.72em;
+  border: 0.13em solid currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)